monitors/index.js

```diff
@@ -30,13 +31,23 @@ const ExpandedERC20 = artifacts.require("ExpandedERC20");
  * @param {String} address Contract address of the EMP.
  * @return None or throws an Error.
  */
-async function run(address, shouldPoll, botMonitorObject, walletMonitorObject, pollingDelay, priceFeedConfig) {
+async function run(
+  address,
+  shouldPoll,
+  botMonitorObject,
+  walletMonitorObject,
+  syntheticPegMonitorObject,
+  pollingDelay,
+  uniswapPriceFeedConfig,
+  medianizerPriceFeedConfig
```

Comment:
nit: we might want to name these a little more generically -- like primary and secondary price feed? Or primary and reference? Not a strong preference, though -- up to you.

Reply:
We _could_ do this but I think that for now, it's probably easier to the name of the reference as specific as possible so there is no opportunity for confusion. Prehaps a good name would be `synethicMarketPriceFeed` and `identifierMarketPriceFeed` but for now, I'm favoring explicit naming.